components/TestResult/vue/TestResult.vue

```diff
@@ -0,0 +1,100 @@
+<script lang="ts" setup>
+import {
+  classes,
+  type TestResultData,
+} from '@cypress-design/constants-testresult'
+import { StatusIcon } from '@cypress-design/vue-statusicon'
+import {
+  IconChevronRightSmall,
+  IconStatusFlaky,
+  IconDocumentModifiedSquareDot,
+  IconDocumentAddedSquarePlus,
+} from '@cypress-design/vue-icon'
+
+defineProps<TestResultData>()
```

Comment:
@elevatebart One thing I'm also remembering now is that we will probably need to add a `hover=true` property or something since the Debug page doesn't actually have a clickable row for these results.

Reply:
This would conditionally add the classes needed for the hover state (i.e. filter, borders, transitions, etc)

Reply:
Should the prop be named "clickable" ?